src/js/routes/services.js

```diff
@@ -1,37 +1,69 @@
 import Router from 'react-router';
-let Route = Router.Route;
+let {Route, Redirect} = Router;
 
+import DeploymentsTab from '../pages/services/DeploymentsTab';
 import ServiceOverlay from '../components/ServiceOverlay';
 import ServicesPage from '../pages/ServicesPage';
+import ServicesTab from '../pages/services/ServicesTab';
 
-let servicesRoutes = {
+let services = {
   type: Route,
   name: 'services',
-  path: 'services/?',
   handler: ServicesPage,
   children: [
     {
       type: Route,
-      name: 'services-detail',
-      path: ':id/?'
+      name: 'services-page',
+      path: '/services/?',
+      children: [
+        {
+          type: Route,
+          name: 'services-all',
+          path: 'all/',
```

Comment:
@kennyt as in we shouldn't? Would be interested to hear suggestions for a better routing scheme - not sure I'm happy with routes like `/services/all/%2Fsome-service`